packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/utils/useLinkTooltips.ts

```diff
@@ -0,0 +1,83 @@
+import type { Editor } from 'tinymce';
+import { useCallback, useEffect } from 'react';
+import { _ } from '@joplin/lib/locale';
+import shim from '@joplin/lib/shim';
+
+const useLinkTooltips = (editor: Editor|null) => {
+	const resetModifiedTitles = useCallback(() => {
+		for (const element of editor.getDoc().querySelectorAll('a[data-joplin-original-title]')) {
+			element.setAttribute('title', element.getAttribute('data-joplin-original-title') ?? '');
+			element.removeAttribute('data-joplin-original-title');
+		}
+	}, [editor]);
+
+	useEffect(() => {
+		if (!editor) return () => {};
+
+		const onMouseOver = (event: MouseEvent) => {
+			let element = event.target as HTMLElement;
+
+			// mouseover events seem to only target the lowest applicable node in the DOM.
+			// If the user's mouse enters <a><strong></strong></a>, the mouseover event will
+			// target the <strong></strong>. As such, the parent nodes need to be checked:
+			let counter = 0;
+			while (element.tagName !== 'A' || !('href' in element)) {
+				element = element.parentElement;
+				counter++;
+				if (!element || counter > 4) {
+					return;
+				}
+			}
+
+			if (!element.hasAttribute('data-joplin-original-title')) {
+				element.setAttribute('data-joplin-original-title', element.title);
+			}
+
+			// Avoid showing internal HREFs for resource and note links.
```

Comment:
```suggestion
			// Avoid showing internal HREFs for note links.
```

Just note links (see above).